### ISSUE TITLE
Contrib Group Application:

### ISSUE THREAD

**Issue by onyxnz** [backdrop-ops/contrib#701]
Hello and welcome to the contrib application process! We're happy to have you :)

**Please indicate how you intend to help the Backdrop community by joining this group**
* **Option 1**: I would like to contribute a project
* Option 2: I would like to maintain a project, but have nothing to contribute at this time
* Option 3: I would like to update documentation and/or triage issue queues

## Based on your selection above, please provide the following information:

**(option 1) The name of your module, theme, or layout**
Webform HTML Textarea

## (option 1) Please note these 3 requirements for new contrib projects:

- [ ] Include a **README.md** file containing license and maintainer information.
      You can use this example: https://raw.githubusercontent.com/backdrop-ops/contrib/master/examples/README.md
- [ ] Include a **LICENSE.txt** file.
      You can use this example: https://raw.githubusercontent.com/backdrop-ops/contrib/master/examples/LICENSE.txt.
- [ ] If porting a Drupal 7 project, Maintain the Git history from Drupal.
Sorry, couldn't figure out how to do this from https://git.drupalcode.org/project/webform_html_textarea

**(option 1 -- optional) Post a link here to an issue in the drupal.org queue notifying the Drupal 7 maintainers that you are working on a Backdrop port of their project**

**Post a link to your new Backdrop project under your own GitHub account (option 1)**

https://github.com/onyxnz/backdrop_webform_html_textarea

**If you have chosen option 2 or 1 above, do you agree to the [Backdrop Contributed Project Agreement](https://github.com/backdrop-ops/contrib#backdrop-contributed-project-agreement)**
**YES**/no

**Comment by BWPanda** [comment]
Hi @onyxnz! Regarding "Maintain the Git history from Drupal", did you see this article?

https://www.tag1consulting.com/blog/how-maintain-contrib-modules-drupal-and-backdrop-same-time-part-2

**Comment by jenlampton** [comment]
Hi @onyxnz -- I often forget to include the git history on my ports, so I wrote up this article on how I "add it back"
https://www.jenlampton.com/blog/how-add-back-git-history-when-you-forgot-clone.  My article is a little simpler than David's :)

**Comment by jenlampton** [comment]
Hi @onyxnz, Are you still interested in joining the contrib group? If you are and you still need help with the commit history, please let us know. We can help :)

**Comment by onyxnz** [comment]
Yes, I am, thanks Jen. But things are a little nuts right now, so will come back to you soon.

**Comment by jenlampton** [comment]
No problem, we'll be here when you have the time :) Thanks for the update!

**Comment by laryn** [comment]
@onyxnz Sorry, I missed this issue earlier and in the meantime I've brought over Webform HTML Textarea from an earlier port some colleagues had done internally:

 - https://github.com/backdrop-contrib/webform_html_textarea

I am certainly open to you being a co-maintainer if you're interested, when you officially join the contrib group.
